agent/php_newrelic.h

```diff
@@ -304,8 +309,9 @@ nr_php_ini_attribute_config_t
                                     */
 
 nrinibool_t custom_events_enabled; /* newrelic.custom_insights_events.enabled */
-nriniuint_t custom_events_max_samples_stored; /* newrelic.custom_events.max_samples_stored */
-nrinibool_t synthetics_enabled;    /* newrelic.synthetics.enabled */
+nriniuint_t custom_events_max_samples_stored; /* newrelic.custom_events.max_samples_stored
+                                               */
+nrinibool_t synthetics_enabled;               /* newrelic.synthetics.enabled */
```

Comment:
This change is unrelated to error grouping. To improve code review experience and avoid distractions caused by unrelated formatting changes, I strongly suggest using `git clang-format` command, which formats only the code that is changed in a commit, not the whole file. You can read more about it [here](https://ortogonal.github.io/cpp/git-clang-format/#:~:text=git%2Dclang%2Dformat%20is%20a,on%20the%20changes%20you%20made).